src/backend/src/prisma/migrations/20231127023813_team_with_user_archived_fixed/migration.sql

```diff
@@ -0,0 +1,15 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `userId` on the `Team` table. All the data in the column will be lost.
+
+*/
+-- DropForeignKey
```

Comment:
can you delete both of these migrations and generate a new one that has all the changes in one migration